src/main/resources/application.yaml

```diff
@@ -1,6 +1,6 @@
 konflux-build-driver:
   quay-repo: "quay.io/redhat-user-workloads-stage/pnc-devel-tenant/pnc-konflux"
-  konflux-processor: "quay.io/redhat-user-workloads/konflux-jbs-pnc-tenant/jvm-build-service/build-request-processor:latest"
+  konflux-processor: "quay.io/redhat-user-workloads/konflux-jbs-pnc-tenant/konflux-tooling:latest"
```

Comment:
@matejonnet I wonder if we should rename this configuration option to e.g. pnc-tooling ? (as its not really a 'processor' any more - that was a legacy name from JBS)

Reply:
For that matter, the `konflux-processor` value is then used to fill out `JVM_BUILD_SERVICE_REQPROCESSOR_IMAGE` but that comes from now https://quay.io/repository/redhat-user-workloads/konflux-jbs-pnc-tenant/konflux-tooling?tab=tags&tag=latest so potentially that also should be renamed (but that is scattered across various places so is more complex to handle)